examples/resources/hpegl_vmaas_network/nsx_t_dhcp_network.tf

```diff
@@ -0,0 +1,31 @@
+# (C) Copyright 2021 Hewlett Packard Enterprise Development LP
+
+resource "hpegl_vmaas_network" "dhcp_net" {
+  name         = "tf_nsx_t_dhcp_network"
+  description  = "DHCP Network create using tf"
+  display_name = "tf_nsx_t_dhcp_network"
+  scope_id     = data.hpegl_vmaas_transport_zone.tf_zone.provider_id
+  cidr         = "193.2.3.1/20" 
+  primary_dns = "8.8.8.8"
+  scan_network = false
+  active       = true
+  allow_static_override = true
+  appliance_url_proxy_bypass = true
+  group {
+    id = "shared"
+  }
```

Comment:
group is not right, please put back in the old way as group_id

Reply:
updated